lib/game.rb

```diff
@@ -7,6 +7,14 @@ def self.all
     @@all
   end
 
+  def find_by(id)
```

Comment:
useful tool for navigating between csv files

Reply:
Oh nice! This will come in handy!